src/content/docs/implementation/setup/content-delivery-network.mdx

```diff
@@ -0,0 +1,605 @@
+|--------------------------------|----------------------------------------------|--------------------------------------------------------|---------------------------------------------------------------|
+| **Routing logic**              | Default to Edge Delivery Services            | Default to Edge Delivery Services                      | Default to Commerce                                           |
+| **Paths routed to Commerce**   | Product images, GraphQL endpoint             | Transactional pages, account page, REST endpoints, etc | All paths except those explicitly                             |
+| **Paths routed to Edge**       | All other paths                              | Product catalog, content pages                         | Homepage                                                      |
+| **VCL snippets focus**         | Default to Edge Delivery Services            | Default to Edge Delivery Services                      | Default to Commerce                                           |
+| **Optimization tips**          | Avoid additional TLS handshake for API calls | Same as full storefront                                | Route resources (JS, CSS) to Edge Delivery Services           |
+| **Common patterns**            | Use CDN as proxy for Catalog Service         | Same as full storefront                                | Move Edge code to subfolder, such as `aem`                    |
+
+:::note
+All paths for the Adobe Commerce Admin should be routed to Commerce.
+:::
+
+### Full storefront
+
+The entire storefront experience is delivered by Edge Delivery Services. Only some paths (for example, images and API calls) are routed to Commerce.
+
+* By default, paths are routed to Edge Delivery Services
+* Product images must be routed to Commerce
+* Commerce GraphQL endpoint must be routed to Commerce
+* CDN must proxy requests to your Catalog Service API endpoint (https://catalog-service.adobe.io/graphql and https://catalog-service-sandbox.adobe.io/graphql).
+
+  :::tip
+  These API endpoints impact largest contentful paint (LCP), so we want to avoid an additional TLS handshake. This optimization has already been [applied](https://github.com/hlxsites/aem-boilerplate-commerce/blob/develop/blocks/product-list-page/product-list-page.js#L15) to the boilerplate, where we use a "custom" version of the Live Search widget to change the API URL used by the widget.
+  :::
+
+### Luma Bridge
+
+The product catalog and content pages are delivered by Edge Delivery Services. Transactional pages (for example, cart, checkout, and account) are delivered by Commerce.
+
+* Includes the same details as the full storefront option, plus the following:
+  * Transactional pages (for example, cart, checkout, and account) are routed to Commerce, depending on the Luma Bridge implementation
```

Comment:
I think "are routed" should be changed to "can be routed", but IDK the details here